front/app/containers/EventsPage/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+// i18n
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import messages from './messages';
+
+// components
+import ContentContainer from 'components/ContentContainer';
+import SectionContainer from 'components/SectionContainer';
+import { Helmet } from 'react-helmet';
+import UpcomingEventsViewer from './UpcomingEventsViewer';
+import PastEventsViewer from './PastEventsViewer';
+
+// styling
+import styled from 'styled-components';
+
+const StyledContentContainer = styled(ContentContainer)`
+  max-width: calc(${(props) => props.theme.maxPageWidth}px - 100px);
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+export default injectIntl<InjectedIntlProps>(({ intl }) => {
+  const upcomingEvents = Array(15)
+    .fill(0)
+    .map((_, i) => i + 1);
+
+  const pastEvents = [];
+
+  return (
+    <>
+      <Helmet>
```

Comment:
Would add the meta info in a separate component (see e.g. IdeaIndexMeta). You need some more info than only the title.